Project-II-Calculator/calculator/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "mathjs": "^5.4.0",
```

Comment:
I've never used this package, but now that I've seen it in action, it's on my list of go-to dependencies :+1: